resources/utility-scripts/test_e2e.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
```

Comment:
Nice tests. Can you add the following at the very top:

```bash
set -Eeuo pipefail
``` 

This sets some bash options to make the execution of scripts more robust, all .sh files in MedCo have it (normally).